styles.css

```diff
@@ -83,3 +98,236 @@ body {
 .img {
   margin-right: 15px;
 }
+
+#section2 {
+  background: #e5e5e5;
+  display: flex;
+  flex-direction: column;
+  padding-top: 114px;
+  align-items: center;
+}

```

Comment:
- [ ] Good work so far, however as mentioned in the requirements, we have to use `grid` for the layout of the whole section, and `flexbox` for the layout of each individual project card, currently you have used flex for the layout of the whole section and grid for the u elements, using grid and flex will give you a lot of freedom in responsive design and both are crucial skills to master, please fix this
![image](https://user-images.githubusercontent.com/97521557/235297548-abec82ee-fd11-45aa-99cd-fd93896abc14.png)